inst/shiny-server/ui/fv_CDP_box.R

```diff
@@ -0,0 +1,163 @@
+# downloadButton('FCEECDF.AUC.Download', label = 'Download the figure'),
+
+
+
+fv_CDP_box <- function(width = 12, collapsible = T, collapsed = T) {
+  box(title = 'Cumulative Difference Plot of Fixed-Budget Function Values',
+      width = width, collapsible = collapsible,
+      solidHeader = TRUE,  collapsed = collapsed,
+      status = "primary",
+      sidebarPanel(
+        width = 3,
+        HTML('Select the budget value.'),
+        textInput('FCEPDF.CDP.Runtime', label = '',
+                  value = ''),
+        hr(),
+        HTML('Select <strong>TWO</strong> IDs.'),
+        selectInput('FCEPDF.CDP.Algs', label = "",
+                    multiple = T, selected = NULL, choices = NULL) %>% shinyInput_label_embed(
+                      custom_icon() %>%
+                        bs_embed_popover(
+                          title = "ID selection", content = "The Cumulative Difference Plot can only compare two IDs at a time.",
+                          placement = "auto"
+                        )
+                    ),
+        hr(),
+        HTML('Select maximization or minimization.'),
+        selectInput('FCEPDF.CDP.Mode', label = '',
+                    choices = c("max.", "min.","Choose one"), selected = "Choose one")%>% shinyInput_label_embed(
+                      custom_icon() %>%
+                        bs_embed_popover(
+                          title = "max. or min.", content = "Choose maximization when higher is better and minimization when lower is better.",
+                          placement = "auto"
+                        )
+                    ),
+        hr(),
+        HTML('Select confidence level.'),
+        selectInput('FCEPDF.CDP.Confidence', label = '',
+                    choices = c(0.9, 0.95, 0.99), selected = 0.95)%>% shinyInput_label_embed(
+                      custom_icon() %>%
+                        bs_embed_popover(
+                          title = "Confidence level", content = "Choose the confidence level of the confidence band.",
+                          placement = "auto"
+                        )
+                    ),
+        hr(),
+        selectInput('FCEPDF.CDP.Format', label = 'select the figure format',
+                    choices = supported_fig_format, selected = supported_fig_format[[1]]),
+        downloadButton('FCEPDF.CDP.Download', label = 'Download the figure'),
+
+      ),
+
+      mainPanel(
+        width = 9,
+        column(
+          width = 12, align = "center",
+          HTML('<p align="left" "font-size:120%;">
+               The cumulative difference plot can be useful when box/violin-plots
+               fail to provide a clear result. Generating this plot has two requirements:
+               </p>
+
+               <p align="left" "font-size:120%;">
+               - Select exactly two algorithms (two IDs)<br>
+               - Select max. (when higher is better) or min. (when lower is better)
```

Comment:
This property can be extracted from the DataSetList object, so it should not be asked to the user